repos/system_upgrade/common/actors/checkunsafepythonpaths/actor.py

```diff
@@ -0,0 +1,24 @@
+from leapp.actors import Actor
+from leapp.libraries.actor.checkunsafepythonpaths import perform_check
+from leapp.models import UnsafePythonPaths
+from leapp.reporting import Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class CheckUnsafePythonPaths(Actor):
```

Comment:
Do not forget to rename all functions, classes, ... as it is not about unsafe python paths but about third party python modules for the target python.